apps/prairielearn/src/pages/instructorRequestCourse/instructorRequestCourse.html.ts

```diff
@@ -224,8 +224,24 @@ function CourseNewRequestCard({ csrfToken }: { csrfToken: string }): HtmlValue {
             <label class="form-label" for="cr-ghuser">GitHub Username (optional)</label>
             <input type="text" class="form-control" name="cr-ghuser" id="cr-ghuser" />
             <small class="form-text text-muted">
-              Providing your GitHub username will allow you to edit course content offline. You do
-              not need to provide this if you would like to use the online web editor.
+              Providing your GitHub username will allow you to edit course content offline or run a
```

Comment:
Do we want to make it clear what this will do (grant access to the repo)? Also edited to shift the last sentence here, though I don't mind if you don't want that.

> Providing your GitHub username will grant you access to your course's GitHub repository and allow you to grant access to other instructors or TAs. You do not need ...

Reply:
Rephrased in 63276424f848ce5a155a68fc3b4732390e7609c9.